File: programmers/python/입문100/15.3_한번만등장한문자/15-3.py
# 1.

def solution(string):
    answer = ''
    for s in string:
        if string.count(f'{s}') == 1:
            answer += s

    return ''.join(sorted(answer))

# 2. 리스트 컴프리헨션

def solution(string):
    return ''.join(sorted([s for s in string if string.count(s) == 1]))

File: programmers/java/기초124/15.3_1로만들기/15_3.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;

        for (int x : num_list) {
            while (x > 1) {
                if (x % 2 == 0) {
                    x /= 2;
                } else {
                    x = (x - 1) / 2;
                }
                answer ++;
            }
        }

        return answer;
    }
}